contracts/HumaniqToken.sol

```diff
@@ -68,7 +69,7 @@ contract HumaniqToken is StandardToken, SafeMath {
         if (tokenCount == 0) {
             return false;
         }
-        if (add(totalSupply, tokenCount) > maxTotalSupply) {
+        if (icoFinished && add(totalSupply, tokenCount) > maxTotalSupply) {
```

Comment:
It's incorrect. The project model requires a possibility to issue new tokens after the end of the ICO. With every new user who passed a bio-identification 100 new tokens must be minted. 
Talking about particular implementation of this feature, what do you think about the following idea: In finilizeMaxTotalSupply() functions we can emit 4 * totalSupply tokens to special address which will be locked for all operations except withdrawing 100 tokens to any other address with 0 balance. Also tokens issued to this address shouldn't be counted in totalSupply.